test/cypress/integration/pages/your-financial-path-to-graduation.js

```diff
@@ -0,0 +1,219 @@
+      page.click( 'Yes' );
+      page.setText( 'costs__tuition-fees', '100000' );
+      page.setText( 'costs__room-board', '50000' );
+      page.setText( 'costs__otherDirect-board', '25000' );
+      cy.get( '[data-financial-item="total_directCosts"]' ).should( 'contain', '$175,000' );
+      page.setText( 'costs__books', '12500' );
+      page.setText( 'costs__transportation', '7500' );
+      page.setText( 'costs__other', '5000' );
+      cy.get( '[data-financial-item="total_indirectCosts"]' ).should( 'contain', '$25,000' );
+      page.setText( 'costs__otherIndirect', '2500' );
+      cy.get( '[data-financial-item="total_costs"]' ).each( el => {
+        cy.wrap( el ).should( 'contain', '$202,500' );
+      } );
+    } );
+    it( 'graduate degree should display total_costs', () => {
+      page.open();
+      page.click( 'Get started' );
+      page.setText( 'search__school-input', 'Harvard' );
+      page.selectProgram( 'type', 'graduate' );
+      page.selectProgram( 'length', '4' );
+      page.selectProgram( 'rate', 'out-of-state' );
+      page.selectProgram( 'housing', 'off-campus' );
+      page.click( 'Next step' );
+      page.click( 'No' );
+      page.setText( 'costs__tuition-fees', '400000' );
+      page.setText( 'costs__room-board', '200000' );
+      page.setText( 'costs__otherDirect-board', '100000' );
+      cy.get( '[data-financial-item="total_directCosts"]' ).should( 'contain', '$700,000' );
+      page.setText( 'costs__books', '50000' );
+      page.setText( 'costs__transportation', '25000' );
+      page.setText( 'costs__other', '12500' );
+      cy.get( '[data-financial-item="total_indirectCosts"]' ).should( 'contain', '$87,500' );
+      page.setText( 'costs__otherIndirect', '10000' );
+      cy.get( '[data-financial-item="total_costs"]' ).each( el => {
+        cy.wrap( el ).should( 'contain', '$797,500' );
+      } );
+      page.click( 'Next step' );
+      page.setText( 'grants__pell', '9000' );
+      page.setText( 'grants__seog', '10000' );
+      page.setText( 'grants__otherFederal', '11000' );
+      page.setText( 'grants__state', '12000' );
+      page.setText( 'grants__school', '13000' );
+      page.setText( 'grants__other', '14000' );
+      page.setText( 'grants__mta', '4500' );
+      page.setText( 'grants__gibill', '15000' );
+      page.setText( 'grants_serviceOther', '16000' );
+      page.setText( 'scholarships__state', '17000');
+      page.setText( 'scholarships__school', '18000');
+      page.setText( 'scholarships__other', '19000');
+      cy.get( '[data-financial-item="total_grantsScholarships"]' ).should( 'contain', '$158,500' );
+      page.click( 'Next step' );
+      page.setText( 'workStudy__workStudy', '50000');
+      cy.get( '[data-financial-item="total_workStudy"]' ).should( 'contain', '$50,000' );
+      page.click( 'Next step' );
+      page.setText('loans__directUnsub', '90000');
+      cy.get( '[data-financial-item="total_fedLoans"]' ).should( 'contain', '$5,442' );
```

Comment:
I'm now seeing this assertion fail here and in the bachelor's degree section. Any idea why that might be?

Reply:
I wonder if it has to do with the program type radio buttons? The snapshots I'm seeing in the Cypress interface do not show them getting selected.

Reply:
I cannot reproduce this issue. The program type radio buttons are hidden behind the school search results.
I have made some minor changes to the `your-financial-path-to-graduation` test.
Please attach a screen snapshot of the Cypress interface showing an assertion failure.

Reply:
Here's what I'm seeing (this is on the Bachelor's section, but it also happens on the Graduate section in the same spot):

![](https://user-images.githubusercontent.com/1044670/102122522-9335a800-3e13-11eb-8c9d-ebd1f915835c.png)

Reply:
I believe this has been resolved by setting loans__directUnsub to 2000.
It seems you cannot have a direct Unsub loan as high as 90000.